client/CMT.cpp

```diff
@@ -1295,9 +1292,12 @@ static void handleEvent(SDL_Event & ev)
 static void mainLoop()
 {
 	SettingsListener resChanged = settings.listen["video"]["fullscreen"];
-	resChanged([](const JsonNode &newState){  CGuiHandler::pushSDLEvent(SDL_USEREVENT, FULLSCREEN_TOGGLED); });
+	resChanged([](const JsonNode & newState)
+	{
+		CGuiHandler::pushSDLEvent(SDL_USEREVENT, FULLSCREEN_TOGGLED);
+	});
 
-	inGuiThread.reset(new bool(true));
+	inGuiThread.reset(new bool (true));

```

Comment:
I think uncrustify would do the same for any of standard types. I suppose it's might be a bug.